addon/components/power-calendar/days.js

```diff
@@ -160,15 +162,18 @@ export default Component.extend({
   buildDay(dayMoment, today, calendar) {
     let id = dayMoment.format('YYYY-MM-DD');
     let momentDate = dayMoment.clone();
+    let isCurrentMonth = momentDate.month() === calendar.center.month();
 
     return {
       id,
       number: momentDate.date(),
       date: momentDate._d,
       moment: momentDate,
+      dow: momentDate.format('ddd').toLowerCase(),
```

Comment:
I'd prefer `dow` to be more clear even if that means it has to be more verbose. `dayName`?